### ISSUE TITLE
[MQ][Issue] Add "Use with Dependency Injection" to the standard README template

### ISSUE THREAD

**Issue by tg-msft** [Azure/azure-sdk-for-net#46393]
### Information

A lot of our usage is driven by ASP.NET customers.  We should add a [standard README section](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/.docsettings.yml#L28-L38) showing a copy/pasteable sample of using a given client with ASP.NET's DI via the extensions present in every library.  I'd put it right after "Authenticate the Client" as an alternative depending on your app model.  We can start simple like [the Aspire docs](https://learn.microsoft.com/en-us/dotnet/aspire/storage/azure-storage-blobs-integration?tabs=dotnet-cli#example-usage) which I've found really helpful for getting started quickly.

**Comment by annelo-msft** [comment]
Good idea!  We are looking at the SCM-based client integration points here right now, and can be tracking these samples as deliverables to come out of that (or work toward) as part of that effort.  @tg-msft do you have specific scenarios you're looking to light up here?

@m-redding FYI
